selfservice/strategy/oidc/provider_config.go

```diff
@@ -94,6 +94,10 @@ type Configuration struct {
 	//
 	// More information: https://openid.net/specs/openid-connect-core-1_0.html#ClaimsParameter
 	RequestedClaims json.RawMessage `json:"requested_claims"`
+
+	// List of values to check audience field of ID Token.
+	// The audience field of ID Token should be equal to one of the items in this list.
+	Audiences []string `json:"audiences"`
```

Comment:
To make the use of this config clear I suggest renaming this to:

```suggestion
	AllowedAudiences []string `json:"allowed_audiences"`
```